cmd/kubeadm/app/apis/kubeadm/validation/validation.go

```diff
@@ -440,9 +440,9 @@ func ValidatePodSubnetNodeMask(subnetStr string, c *kubeadm.ClusterConfiguration
 		// the pod subnet mask needs to allow one or multiple node-masks
 		// i.e. if it has a /24 the node mask must be between 24 and 32 for ipv4
 		if maskSize > nodeMask {
-			allErrs = append(allErrs, field.Invalid(fldPath, podSubnet.String(), "pod subnet size is smaller than the node subnet size"))
+			allErrs = append(allErrs, field.Invalid(fldPath, podSubnet.String(), fmt.Sprintf("pod subnet size %d is smaller than the node subnet size %d", maskSize, nodeMask)))
```

Comment:
Emm.. `mask size` and `subnet size` are different. The bigger the mask, the smaller the subnet.
```suggestion
			allErrs = append(allErrs, field.Invalid(fldPath, podSubnet.String(), fmt.Sprintf(" the size of pod subnet with mask %d is smaller than the size of node subnet with mask %d", maskSize, nodeMask)))
```